chapters/functions.tex

```diff
@@ -1341,17 +1336,17 @@ \subsection{Using the Derivative Annotation}\doublelabel{using-the-derivative-an
   ...
 \end{lstlisting}
 the equation
-
-(...,y(t),...)=foo0(...,x(t),b,...);
-
+\begin{lstlisting}[language=modelica]
```

Comment:
I wouldn't view this as Modelica-code, due to y(t). Thus I would propose to keep as before for now.

Reply:
OK, I put it in an `align*` environment instead.  How about that?

Reply:
Sounds good.